### PR TITLE
router-v2: add uploading icon to tab bar

### PR DESCRIPTION
**Pull request by jakob223**
Issue: KBFS-4017

### CODE REVIEW

File: shared/router-v2/tab-bar/container.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const mapStateToProps = state => ({
   _badgeNumbers: state.notifications.navBadges,
   fullname: TrackerConstants.getDetails(state, state.config.username).fullname || '',
   isWalletsNew: state.chat2.isWalletsNew,
+  uploading: state.fs.uploads.syncingPaths.size > 0 || state.fs.uploads.writingToJournal.size > 0,
```
Comment (cjb) [inline shared/router-v2/tab-bar/container.js]: FWIW I prefer `.count()` to `.size` -- if the immutable sequence is lazy, size doesn't force evaluation and returns undefined, whereas `.count()` does.

**Comment by jakob223** [comment]
@cjb was that a review with one nit, or a comment without review? (i.e. should I merge or tag react hackers again?)

File: shared/router-v2/tab-bar/index.desktop.js [hunk 2]
```diff
@@ -141,7 +141,16 @@ class TabBar extends React.PureComponent<Props, State> {
                   style={styles.tab}
                 >
                   <Kb.Box2 className="tab-highlight" direction="vertical" fullHeight={true} />
-                  <Kb.Icon className="tab-icon" type={data[t].icon} sizeType="Big" />
+                  <Kb.Box style={styles.iconBox}>
```
Comment (jzila) [inline shared/router-v2/tab-bar/index.desktop.js]: `Box2` please.

**Comment by jakob223** [comment]
Bump @jzila

**Pull request merged**